tree/20_cfe_basics/cfengine/paths.cf

```diff
@@ -101,12 +104,14 @@ bundle common paths
     _have_bin_systemctl::
       "path[systemctl]"      string => "/bin/systemctl";
     !_have_bin_systemctl::
-      "path[systemctl]"      string => "/usr/bin/systemctl";
+      "path[systemctl]"      string => "/bin/systemctl";
 
     linux::
       "path[lsattr]"        string => "/usr/bin/lsattr";
       "path[tar]"           string => "/bin/tar";
-      "path[chkconfig]"     string => "/sbin/chkconfig";
```

Comment:
can't we have chkconfig on other system than redhat-like, debian-like, and suse?

Reply:
Added in https://github.com/Normation/ncf/commit/78b8e471648d18c83b826ddc6fbf80d08675f388 but it does not seem necessary after the changes in the generic method.